registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -92,6 +92,7 @@
     "sha256:08f3b9ca2145435e59ae7619bfe07c30ec7a799ce1afa5320ced3ccabd03b81b": ["v1.11.0-alpha.0"]
     "sha256:a8cb1c2ac36c416fa15e8029bbfaf21c8653973141546f0a318e789c9cbd7783": ["v1.11.0-alpha.1"]
     "sha256:bdc1b5e0ff8d93adc37338e14c5ed7a4fca172f0e30e19281da50fbf1b6b2043": ["v1.11.0-alpha.2"]
+    "sha256:a316a24bd7d2e61cae26e546e23889ef85e42d6d4e24bdb0a885ec9e3194b78a": ["v1.11.0-beta.0"]
```

Comment:
SHA `a316a24bd7d2` matches cluster-api-controller v1.11.0-beta.0 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/cluster-api-controller@sha256:a316a24bd7d2e61cae26e546e23889ef85e42d6d4e24bdb0a885ec9e3194b78a/details?inv=1&invt=Ab21Sw). 👍🏻